app/src/main/res/layout/activity_create_event.xml

```diff
@@ -1,182 +1,191 @@
-        android:layout_height="48dp"
-        android:ems="10"
-        android:gravity="start|top"
-        android:hint="Team"
-        android:inputType="textMultiLine"
-        tools:layout_editor_absoluteX="39dp"
-        tools:layout_editor_absoluteY="370dp" />
-
-    <EditText
-        android:id="@+id/etDetails"
-        android:layout_width="300dp"
-        android:layout_height="72dp"
-        android:ems="10"
-        android:gravity="start|top"
-        android:hint="Details About Event"
-        android:inputType="textMultiLine"
-        tools:layout_editor_absoluteX="39dp"
-        tools:layout_editor_absoluteY="289dp" />
-
-    <EditText
-        android:id="@+id/etEventCode"
-        android:layout_width="115dp"
-        android:layout_height="41dp"
-        android:ems="10"
-        android:gravity="start|top"
-        android:hint="Event Code"
-        android:inputType="textMultiLine"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.702"
-        tools:layout_editor_absoluteX="39dp" />
-
-    <TextView
-        android:id="@+id/tvCreateEvent"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="16dp"
-        android:text="Create An Event"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
-
-    <TextView
-        android:id="@+id/tvSkillLevel"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Skill Level"
-        tools:layout_editor_absoluteY="234dp" />
-
-    <TextView
-        android:id="@+id/tvAllowPlusOnes"
-        android:layout_width="64dp"
-        android:layout_height="35dp"
-        android:text="Allow Plus Ones"
-        tools:layout_editor_absoluteX="115dp"
-        tools:layout_editor_absoluteY="231dp" />
-
-    <TextView
-        android:id="@+id/tvAllowSpectators"
-        android:layout_width="94dp"
-        android:layout_height="35dp"
-        android:text="Allow Spectators"
-        tools:layout_editor_absoluteX="231dp"
-        tools:layout_editor_absoluteY="229dp" />
-
-    <EditText
-        android:id="@+id/edDate"
-        android:layout_width="179dp"
-        android:layout_height="45dp"
-        android:ems="10"
-        android:hint="Date"
-        android:inputType="date"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.701"
-        tools:layout_editor_absoluteX="168dp" />
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        >
+
+        <TextView
```

Comment:
Generally you don't want to sue a TextView for this because it doesn't look like it's clickable.  Best thing to do is probably to have a place where the time is displayed and a button to change it.  Or make it an edit text but override the onClick so that you show the picker instead of allowing them to edit it.  Then the user knows it's something they should click on.